now.json

```
@@ -10,5 +10,12 @@
       "FIREBASE_REGION": "us-central1",
       "FIREBASE_API_KEY": "AIzaSyCRpaEkvCDjybq6XPQOCRMKsiBO9mgmcLE"
     }
-  }
+  },
+  "routes": [
+    {
+      "src": "/deep-links/*",
```

This is a regex, so here we're looking for zero or more `/`. That's not what we want.

Maybe something like this?:

```suggestion
      "src": "/deep-links/.*",
```